archive/tar_windows.go

```diff
@@ -0,0 +1,9 @@
+// +build windows
+
+package archive
+
+func setUmask(newMask int) (oldMask int) {
+	panic("Not implemented on Windows")
```

Comment:
We intentionally decided to defer this, since it would require a bit of thinking, until further Windows work is done. See description of PR for details.